src/utils/contentful/contentful.ts

```diff
@@ -1,15 +1,23 @@
 import { createClient, Environment } from 'contentful-management';
+import { createClient as createCDAClient } from 'contentful';
 import * as dotenv from 'dotenv';
 
 // load dotenv config
 dotenv.config();
 
-const client = createClient({
+export const CONTENTFUL_LIMIT = 25;
+
+export const cdaClient = createCDAClient({
+	space: `${process.env.CONTENTFUL_SPACE_ID}`,
+	accessToken: `${process.env.CONTENTFUL_DELIVERY_API_TOKEN}`,
```

Comment:
env are default strings. Was this necessary?

Reply:
Typescript is not recognizing the env vars in either `.env` or `serverless.yml` file, so it thinks those vars are `string | undefined`

Reply:
https://stackoverflow.com/questions/45194598/using-process-env-in-typescript